app/src/controllers/log/index.js

```diff
@@ -17,3 +23,14 @@ export {
   previousLogLinkSelector,
   retryLinkSelector,
 } from './selectors';
+export {
```

Comment:
Are all this functions should be exported? They use each other and I think that the list should look like this:
```
  getWithAttachments,
  setWithAttachments,
  setLogLevel,
  getLogLevel,
  getLogViewMode,
  setLogViewMode,
```

Reply:
Fixed. But also we need `getLogLevelById` util to take corresponding log level object with `id` and `label`